practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -146,6 +211,7 @@ const FormValidation = (props) => {
                 name='gender'
                 value='Male'
```

Comment:
create constant gender but not use

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/d2194164-75d2-46bd-ad69-59ddbfa85bcc)